src/client/views/index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<!--
+To change this license header, choose License Headers in Project Properties.
+To change this template file, choose Tools | Templates
+and open the template in the editor.
+-->
+<html ng-app="interview_coding">
+    <head>
+        <% if (title) { %>
+        <title><%= title %></title>
+        <% } %>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+        <!--        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>-->
+        <script src="node_modules/angular/angular.min.js" type="text/javascript"></script>
+        <script src="app/app.js" type="text/javascript"></script>
+        <!--Services-->
+        <script src="app/services/services.js" type="text/javascript"></script>
+        <!--Controllers-->
+        <script src="app/controllers/controllers.js" type="text/javascript"></script>
+        <!--Directives-->
+        <script src="app/directives/directives.js" type="text/javascript"></script>
+    </head>
+    <body>
+        <div ng-controller="show_fav_stuff_list">
+
+            <div class="container">
+                <div>
+                    <table class="table-bordered">
+                        <thead>
+                        <th>Favorite Stuff Name</th>
+                        <th>Favorite Stuff List</th>
+                        </thead>
+                        <tbody>
+                            <tr ng-repeat="sn in fav_stuff track by $index">
+                                <td valign="top" style="padding:10px;">
+                                    <input type="text" ng-disabled="!sn[is_editing]" ng-model="sn.fav_stuff_name">
+                                </td>
+                                <td valign="top" style="padding:10px;">
+                                    <div style="margin:5px 5px;" ng-repeat="sl in sn.fav_stuff_list track by $index">
+                                        <input ng-disabled="!sn[is_editing]" type="text" ng-model="sn.fav_stuff_list[$index]"><button ng-disabled="!sn[is_editing]" type="button" ng-click="delete_element(sn.fav_stuff_list, $index)" class="btn btn-danger">Delete</button>
```

Comment:
Would be nice if markup for nested elements was separated in distinct lines and maybe attributes and directives too, so that the LOC won't be too long.